lib/graphql/schema.flow.js

```diff
@@ -30,7 +30,6 @@ export type User = {|
   __typename?: 'User',
   email: $ElementType<Scalars, 'String'>,
   createdAt: $ElementType<Scalars, 'DateTime'>,
-  vatCutoffLine?: ?$ElementType<Scalars, 'DateTime'>,
```

Comment:
I saw that this deleted line is linked to your previous prs from BE and webapp 👍

Reply:
yes its the reason why I do this update.
Problem is that there is a test on BE that uses the sdk to request `vatCutoffline`.
So the test from this PR always fails https://github.com/kontist/backend/pull/9220 so I update the sdk first. 
Kind of circular depenedency between the test and sdk 😅